server/app/views/applicant/ProgramFiltersFragment.html

```diff
@@ -11,53 +11,57 @@
       name="applicantId"
       th:value="${applicantId.orElse('')}"
     />
-    <fieldset class="usa-fieldset grid-row grid-gap border-0 margin-y-3">
-      <legend
-        class="usa-legend margin-bottom-105 cf-prose-h4"
-        th:text="#{label.programFilters.v2}"
-      ></legend>
-      <th:block th:each="categoryOption : ${categoryOptions}">
-        <div
-          class="usa-checkbox desktop:grid-col-4 tablet:grid-col-6 grid-col-12"
-        >
-          <input
-            th:id="${'ns-check-category-' + #strings.replace(categoryOption, ' ', '-')}"
-            type="checkbox"
-            name="categories"
-            th:value="${categoryOption}"
-            class="usa-checkbox__input"
-          />
-          <label
-            th:for="${'ns-check-category-' + #strings.replace(categoryOption, ' ', '-')}"
-            th:text="${categoryOption}"
-            class="usa-checkbox__label usa-prose"
-          ></label>
-        </div>
-      </th:block>
-      <ul class="usa-button-group margin-top-105">
-        <li class="usa-button-group__item">
-          <button
-            th:text="#{button.applySelections}"
-            id="filter-submit"
-            type="submit"
-            class="usa-button"
+    <fieldset class="usa-fieldset margin-y-3">
+      <div class="grid-row">
```

Comment:
Differences:
- wrapping each part (title, filters, buttons) on a `grid-row`
-  `grid-gap` was moved from fieldset to the filters row
- `border-0` was removed from fieldset. I think this had no effect